internal/workflow/processing.go

```diff
@@ -6,9 +6,9 @@
 package workflow
 
 import (
+	"crypto/rand"
 	"errors"
 	"fmt"
-	"math/rand"
```

Comment:
I think this (load distribution) was a legitimate use of math/rand, i.e. we don't need cryptographic security. You could ignore this warning with a "nosec" comment.

Reply:
@sevein I've switched back to `math/rand` and added a "nosec" comment.